### ISSUE TITLE
Hello man, I found a new problem, I can't open HLS video in Safari

### ISSUE THREAD

**Issue by kira2015** [balvinderz/video_player_web_hls#13]
The Console shows `videoPlayerController.initialize()` all the time

**Comment by balvinderz** [comment]
<img width="1434" alt="Screenshot 2021-07-01 at 7 45 59 PM" src="https://user-images.githubusercontent.com/30950893/124139141-f67fe280-daa4-11eb-9f2a-b6c5842d5321.png">
works on safari too

**Comment by balvinderz** [comment]
which version of video_player_web_hls are you using

**Comment by kira2015** [comment]
video_player_web_hls: ^0.1.11+2
video_player : 2.1.6
Safari : 14.1

**Comment by kira2015** [comment]
console 
![image](https://user-images.githubusercontent.com/14231117/124209962-2f5b9e00-db1d-11eb-8663-8a750bc0b535.png)

**Comment by balvinderz** [comment]
Can you give me the link that you are trying to play

**Comment by kira2015** [comment]
https://github.com/flutter/flutter/issues/47030
My problem is the same as his，Safari will reappear when running on iOS

**Comment by atgnatus** [comment]
For me, HLS video would not play on safari on any platform.  iOS and Android apps were fine as was web video playback on edge, chrome, firefox.

There were 2 issues.  First, the failed to load resources so I changed to this:

<script src="https://cdn.jsdelivr.net/npm/hls.js@latest/dist/hls.min.js"  type="application/javascript"></script>
  
Second, safari would not play video with flutterWebRenderer set to canvaskit so I changed to "html".  Video played everywhere after that.
